src/component/organism/navbar/navbar.tsx

```diff
@@ -17,8 +17,8 @@ interface NavBarPropsModel {
 
 export const Navbar: FC<NavBarPropsModel> = ({ categories }) => {
   const [state, dispatch] = useReducer(reducer, initState);
-  const { contextValue } = useCartContext();
-  
+  const context = useCartContext();
```

Comment:
Find better name for that? What if in the future you need to use other contexts? How would you call them?